rustls-libssl/src/error.rs

```diff
@@ -215,6 +215,12 @@ impl From<Error> for () {
     }
 }
 
+impl From<Error> for crate::entry::SSL_verify_cb {
+    fn from(_: Error) -> crate::entry::SSL_verify_cb {
+        None
+    }
+}
```

Comment:
I'm not sure I understand why this `From` gets added here. I'm probably overlooking something small :thinking:

Reply:
This is so the `try_clone_arc!` inside `_SSL_CTX_get_verify_callback` can raise an error if the pointer it gets is null.

Reply:
:bulb: